src/classes/UTIL_AbstractCommon_BATCH_TEST.cls

```diff
@@ -135,22 +135,18 @@ public class UTIL_AbstractCommon_BATCH_TEST {
     /*******************************************************************************************************************
      * @description Test Batch Execution when additional where clause is present (non LDV)
      */
-    // @isTest
-    // TODO - Fix this error: 
-    // System.UnexpectedException: No more than one executeBatch can be called from within a test method. 
-    // Please make sure the iterable returned from your start method matches the batch size, resulting in 
-    // one executeBatch invocation.
+    @isTest
     public static void executeAddtlWhereClause() {
         List<Account> accs = UTIL_UnitTestData_TEST.createMultipleTestAccounts(TEST_ACCOUNT_CNT, null);
         insert accs;
 
         Test.startTest();
         UTIL_AbstractCommon_MOCK tab = new UTIL_AbstractCommon_MOCK(10, 'Id = \'' + accs[0].Id + '\'');
-        tab.setTestChunkSize(10);
-        tab.setTestMaxNonLDVSize(10);
+        tab.setTestChunkSize(UTIL_AbstractCommon_BATCH.DEFAULT_LDV_CHUNK_SIZE);
```

Comment:
Rather than `tab` as the var name for the instance, how about `mockAbstractCommon`?